src/iiif/utils_azure.py

```diff
@@ -65,6 +65,29 @@ def store_object_on_storage_account(local_zip_file_path, filename):
     return blob_client, blob_service_client
 
 
+def store_blob_on_storage_account(storage_container, blob_name, blob):
```

Comment:
Deze is een beetje een dubbel van store_object_on_storage_account() hierboven. Ik zou de functies mergen zodat er maar één over blijft. Dat houdt de boel schoon.

Reply:
Ik had dat gevoel ook maar het dient wel echt een ander doel. De data kan nu als een stream van de file meteen verzonden worden. Anders wordt het eerst in het werkgeheugen geladen.